scripts/tests/api_compare/filter-list-offline

```diff
@@ -25,9 +25,6 @@
 !Filecoin.StateCall
 # These methods don't make sense in the context of an offline node
 !Filecoin.MinerCreateBlock
-# The estimation is inaccurate only for offline RPC server, to be investigated: https://github.com/ChainSafe/forest/issues/4555
```

Comment:
Wow, so it's resolved automagically?!

Reply:
Indeed, Forest is self-healing.

Reply:
And if so, does it close https://github.com/ChainSafe/forest/issues/4555?